lib/publiccloud/instance.pm

```diff
@@ -39,38 +39,44 @@ Timeout can be set by C<timeout> or 90 sec by default.
 C<<proceed_on_failure=>1>> allows to proceed with validation when C<cmd> is
 failing (return non-zero exit code)
 By default, the command is passed in single quotes to SSH.
-To avoid quoting us C<<no_quote=>1>>.
+To avoid quoting use C<<no_quote=>1>>.
 With C<<ssh_opts=>'...'>> you can overwrite all default ops which are in
 C<<$instance->ssh_opts>>.
 Use argument C<username> to specify a different username then
 C<<$instance->username()>>.
 Use argument C<rc_only> to only check for the return code of the command.
+Use argument C<cmd_timeout> to specify a soft timeout for the command. This must be lower than C<timeot>-1 otherwise it will be set to that value.
```

Comment:
```suggestion
Use argument C<cmd_timeout> to specify a soft timeout for the command. This must be lower than C<timeout>-1 otherwise it will be set to that value.
```